pkg/webui/console/components/device-data-form/index.js

```diff
@@ -187,16 +188,13 @@ class DeviceDataForm extends Component {
           name="join_server_address"
           component={Input}
         />
-        <Form.Field
+        <JoinEUIPrefixesInput
           title={sharedMessages.joinEUI}
           name="ids.join_eui"
-          type="byte"
-          min={8}
-          max={8}
           description={m.joinEUIDescription}
           required
           disabled={update}
-          component={Input}
+          select={!update}
```

Comment:
Nitpick: just `select` is quite ambiguous. I'd prefer something like `showPrefixes` as prop name.